app/assets/stylesheets/style.scss

```diff
@@ -332,7 +333,9 @@ a:hover {
     font-size: $mobile-font-size;
   }
 
-  .subscribe input[type="email"] {
+  .subscribe 

```

Comment:
this line should be `.subscribe {` instead of `.subscribe` to nest, once that's done everything is good 👍